apps/trading/pages/liquidity/[marketId].page.tsx

```diff
@@ -69,9 +69,13 @@ const LiquidityPage = ({ id }: { id?: string }) => {
       <div className="h-full grid grid-rows-[min-content_1fr]">
         <Header
           title={
-            <button onClick={() => push(`/markets/${marketId}`)}>{`${name} ${t(
-              'liquidity provision'
-            )}`}</button>
+            <button
+              className="hover:text-neutral-500 dark:hover:text-neutral-300 sm:text-lg md:text-xl lg:text-2xl "
```

Comment:
Chore (potentially blocking): Is this a one-off button style that won't be repeated? This kind of custom styling on a button element makes consistency and repeatability a challenge as the apps evolve. If there's any chance it'd be repeated, either in the trading app or if you foresee a use case on one of the other apps, a new button variant may be a better option.

Reply:
Yeah I think this could be reused as a back button whenever a market goes to a new page rather than shows something in a panel. For now we only have the liquidity page but there may be more in the future. I think it'd be only used in Console though. So would a new button style in the UI toolkit be the best place for it to go, or somewhere else?

Reply:
I think a new button style in the UI toolkit's still the best place yeah. The only thing we want to avoid is putting elements in there that we don't currently have a use for, but as this is already being used and sounds like it'll be more useful in future, it should be good :)

Reply:
Have created a new Link variant rather than button as I think in this instance it's a navigational link, rather than an action button. Let me know if that looks OK! https://github.com/vegaprotocol/frontend-monorepo/pull/1396/commits/c3ab0c322c8c7ca525deebad2e52ac3a5821ffb5